frontend/shared-components/response-textarea/response-textarea.tsx

```diff
@@ -12,13 +12,14 @@ export default function ResponseTextArea({ isSubmitted }: ResponseTextAreaProps)
   
   const [charCount, setCharCount] = useState(0)
   const [isValid, setIsValid] = useState(!isSubmitted)
-
-  function handleChange($event: FormEvent<HTMLTextAreaElement>): void {
-    const currCharCount: number = $event.target.value.length
-
-    if (isSubmitted) setIsValid(currCharCount >= charMin)
-    setCharCount(currCharCount)
+  
+  function handleTextareaChange({ target: { value }}: FormEvent<HTMLTextAreaElement>): void {
+    if (isSubmitted) setIsValid(value.length >= charMin)
+    setCharCount(value.length)
   }
+  
+
+  useEffect(() => { if (isSubmitted) setIsValid(charCount >= charMin) }, [isSubmitted])
```

Comment:
This `useEffect` seems unnecessary. Did you run into some sort of race condition or timing issue without it?

Reply:
The useEffect is needed so that the validation is only run once the user tries to submit the form - we don't want error messaging to show up before they've completed

Reply:
But we're _not_ preventing validation after form submission(?) We're enabling it - we're preventing validation _before_ submission

The logic so far is basically 
1) the user lands on an empty form by default & can fill it out, we don't want to validate their inputs yet as they haven't clicked "Submit" (also in accordance with POLA)
2) the user clicks submit which changes `isSubmitted` to true
3) **if there are any errors** we want to stay on the form & display said errors (as navigating away from a form in error would actually violate POLA)
4) **if there _aren't_ any errors** we would then navigate away from the page, submitting the data, but this logic isn't included yet as we have nowhere to actually send said data

I've run through it to make sure it works! You could pull my branch & serve it up to see - or I can get a screen recording later on too if that's helpful!

Reply:
Ok, I see what you're going for. My apologies, I got some really bad news today and I'm clearly not in a headspace where I can review code with the required clarity. Withdrawing my objection to this.